superchic.sh

```diff
@@ -0,0 +1,59 @@
+package: SuperCHIC
+version: "%(tag_basename)s%(defaults_upper)s"
+tag: "alice/v2.04"
+source: https://github.com/hcab14/SuperCHIC.git
+requires:
+ - lhapdf
+build_requires:
+ - curl
+ - "GCC-Toolchain:(?!osx)"
+---
+#!/bin/bash -e
+
+rsync -a --exclude '**/.git' $SOURCEDIR/ ./
+
+patch makefile <<EOF
+8c8
+< LHAPDFLIB = /usr/local/Cellar/lhapdf/6.1.5/lib
+---
+> LHAPDFLIB = ${LHAPDF_ROOT}/lib
+
+EOF
+
+mkdir -p obj
+
+make ${JOBS+-j $JOBS}
+
+# the makefile does not contain "install" -> manual installation
+cp -R bin $INSTALLROOT
+
+# install lhapdf sets MMHT2014*
+lhapdf update
+
+PDFSETS=`echo MMHT2014{lo68cl,lo_asmzsmallrange,nlo68cl,nlo68cl_nf{3,4},nlo68cl_nf4as5,nlo68clas118,nlo68clas118_nf{3,4},nlo68clas118_nf4as5,nlo_asmzlargerange,nlo_asmzsmallrange,nlo_asmzsmallrange_nf{3,4},nlo_mbrange_nf{3,4,5},nlo_mcrange_nf{3,4,5},nloas118_mbrange_nf{3,4,5},nloas118_mcrange_nf{3,4,5},nnlo68cl,nnlo68cl_nf{3,4},nnlo68cl_nf4as5,nnlo_asmzlargerange,nnlo_asmzsmallrange,nnlo_asmzsmallrange_nf{3,4},nnlo_mbrange_nf{3,4,5},nnlo_mcrange_nf{3,4,5}}`
```

Comment:
Use `$()` instead of backticks (we use bash, we are not POSIX-compliant anyways).